kyo-core/shared/src/main/scala/kyo/ios.scala

```diff
@@ -1,25 +1,28 @@
 package kyo
 
+import IOs.internal.*
 import core.*
-import core.internal.*
-import iosInternal.*
 import java.util.concurrent.atomic.AtomicReference
 import scala.annotation.tailrec
 import scala.util.*
 import scala.util.control.NonFatal
 
-sealed trait IOs extends Effect[IOs]:
+opaque type IOs <: Defers = Defers & SideEffects
 
-    private val tag = Tag[IOs]
-
-    val unit: Unit < IOs = ()
+object IOs:
 
     inline def apply[T, S](
         inline f: => T < (IOs & S)
     ): T < (IOs & S) =
-        new KyoIO[T, S]:
-            def apply(v: Unit, s: Safepoint[IOs & S], l: Locals.State) =
-                f
+        SideEffects.suspend[Unit, T, Defers & S]((), _ => f: T < (Defers & SideEffects & S))
```

Comment:
I don't love the `SideEffects` name, but I'm not sure of anything better.

Reply:
Yeah, Kyo typically doesn't use composite names. Maybe `Unsafes`? It doesn't seem to sound good, though